cilium/cmd/monitor.go

```diff
@@ -59,40 +71,103 @@ var (
 		SampleType:   bpf.PERF_SAMPLE_RAW,
 		WakeupEvents: 1,
 	}
+	eventTypeIdx = -1 // for integer comparison
+	eventType    = ""
+	eventTypes   = map[string]int{
+		"drop":    bpfdebug.MessageTypeDrop,
+		"debug":   bpfdebug.MessageTypeDebug,
+		"capture": bpfdebug.MessageTypeCapture,
+	}
+	fromSource = uint16(0)
+	toDst      = uint32(0)
+	related    = uint32(0)
 )
 
 func lostEvent(lost *bpf.PerfEventLost, cpu int) {
-	fmt.Printf("Lost %d events\n", lost.Lost)
+	fmt.Printf("CPU %02d: Lost %d events\n", lost.Lost, cpu)
```

Comment:
The ordering seems incorrect here - it should be:
`fmt.Printf("CPU: %02d: Lost %d events\n", cpu, lost.Lost)`